addon/server.js

```diff
@@ -186,7 +193,7 @@ export default class Server {
       this.loadFactories(config.factories);
     } else if (!this.isTest() && hasDefaultScenario) {
       this.loadFactories(config.factories);
-      config.scenarios.default(this);
+      this.loadScenario();
```

Comment:
Let's call this `populateDatabase` or `seedDatabase` since scenario has a more specific meaning (most folks will think of the `/scenarios` dir).